consumer/consumer.py

```diff
@@ -20,6 +18,10 @@ def messages(self, topic = None):
 
 
 if __name__ == "__main__":
+    try:
+        KAFKA_HOST = os.environ['KAFKA_HOST']
+    except KeyError as err:
+        print("Please set KAFKA_HOST environment variable")
```

Comment:
You still need to rethrow as this is an exception we're not able to (or are not going to) handle